### DISCUSSION TITLE
Missing data after October 1st

### DISCUSSION THREAD

**Discussion by larronni** [flobz/psa_car_controller#969] [General]
I have experienced since October 1st that the position on my DS3 is not updated anymore. Could this relate to Eevee "shut down" the free account? Do anyone else have this issue?
It looks like outside temp, ODO meter,  position, driving state, and speed is missing. But everything else is working...

**Comment by naseweiss99** [comment]
same here - with an eC4.
seems to be like it was before EEVVEE enabled the additional data.  But ODO is still updated - as before.
